client/app/domains/domains.html

```diff
@@ -91,16 +92,32 @@ <h1 data-i18n-static="domains_dashboard_title"></h1>
                                     </td>
                                     <td data-ng-bind="domain.creation|date:'mediumDate'"></td>
                                     <td class="text-center">
-                                        <span data-ng-if="domain.whoisFields.length > 0"
-                                              data-i18n-static="common_activated"></span>
-                                        <span data-ng-if="domain.whoisFields.length === 0"
-                                              data-i18n-static="common_desactivated"></span>
+                                        <span class="label label-success"
+                                              data-i18n-static="common_activated"
+                                              data-ng-if="domain.whoisFields.length > 0"></span>
+                                        <span class="label label-danger"
+                                              data-i18n-static="common_desactivated"
+                                              data-ng-if="domain.whoisFields.length === 0"></span>
                                     </td>
                                     <td class="text-center">
-                                        <span data-ng-if="domain.dnssecSupported && domain.dnssecStatus !== null"
-                                              data-ng-bind="::i18n['domains_dashboard_table_dnssec_status_' + domain.dnssecStatus]"></span>
-                                        <span data-ng-if="!domain.dnssecSupported"
-                                              data-i18n-static="common_not_supported"></span>
+                                        <span class="label"
+                                              data-ng-bind="::i18n['domains_dashboard_table_dnssec_status_' + domain.dnssecStatus]"
+                                              data-ng-class="{'label-success': domain.dnssecStatus === 'ENABLED', 'label-danger': domain.dnssecStatus === 'DISABLED', 'label-info': domain.dnssecStatus !== 'ENABLED' && domain.dnssecStatus !== 'DISABLED'}"
```

Comment:
It looks like `domain.dnssecStatus` could be a boolean and the text just above could be computed according to the truthness or falsness of the statement